### PR TITLE
Helps doctors follow the SOP

### PR DESCRIPTION
**Pull request by That0nePerson**
## About The Pull Request
Adds the anesthetic tank holder, which can hold an internals tank and give someone a mask connected to that tank when click-dragged onto them. This allows for medical to have an easy way to anesthetize people, without having to strip them of many pieces of clothing and attach a tank to their back. Click-drag to attach the mask, click to retract mask, alt-click to remove the tank. Can be made out of an IV drip by using a screwdriver on it while having a breath mask in hand.

Adds the latex glove box, which works like the paper bin but for gloves. On click, puts a pair of gloves into your hands. This allows for quick disposal and re-equipping of latex gloves, keeping surgery sterile, and allowing doctors to "wash their hands" by simply changing gloves.

Adds a few sinks to any operating rooms that do not have them in order to wash tools, as well as a box of latex gloves and an anesthetic tank holder per surgery table. 

## Why It's Good For The Game
Allows medical to more easily follow the SOP without it being a huge hassle, and increases the availability of anesthetic. Increased anesthetic use allows for more successful surgeries, better RP, and more traitor opportunities on sleeping patients.

## Changelog
:cl:
add: Added anesthetic tank holder! Portable tank holder allowing for much easier anesthetization of patients.
add: Added quick latex glove box! Allows for quick switching of gloves, to keep your hands sterile.
add: Added sinks to a few operating rooms that did not have them, for surgery tool washing.
/:cl:

### CODE REVIEW

**Comment by Sinmero** [comment]
Feels like this could be weaponized.

**Comment by Effemu** [comment]
cant wait to get valided for following SOP

**Comment by Rukofamicom** [comment]
Definitely feels like it could be weaponized, but still seems like a nice simplification over the current procedure. 
Give it a long use-time so players can't easily just sleepify people indefinitely.

**Comment by NickEmbased** [comment]
based :mask:

**Comment by ItsTisme** [comment]
I pray to every god in the space omniverse that people wont make this into a "cant have nice things"

**Comment by eeSPee** [comment]
> Feels like this could be weaponized.

And I love it.

**Comment by That0nePerson** [comment]
yeah it can be weaponized if you are good enough, but I did think that 3 seconds was a bit short
and increased anesthetic use still allows for a TON of traitor stuff

**Comment by Crossedfall** [comment]
Solid idea

**Comment by DatBoiTim** [comment]
To be fair, anything can be weaponized

File: code/game/machinery/iv_drip.dm [hunk 1]
```diff
@@ -220,11 +221,27 @@
...
+	. = ..()
+	if(user.is_holding_item_of_type(/obj/item/clothing/mask/breath) && can_convert)
+		visible_message("<span class='warning'>[user] attemps to covert the [src] into a anesthetic tank holder.</span>", "<span class='notice'>You attempt to convert the [src] into a anesthetic tank holder.</span>")
+		if(!do_after(user, 100, FALSE, src))
+			to_chat(user, "<span class='warning'>You fail to convert the [src]!</span>")
+			return
+		if(!user.is_holding_item_of_type(/obj/item/clothing/mask/breath)) // Check after the do_after as well
+			return
+		visible_message("<span class='warning'>[user] converts the [src] into a anesthetic tank holder.</span>", "<span class='notice'>You convert the [src] into a anesthetic tank holder.</span>")
+		qdel(user.is_holding_item_of_type(/obj/item/clothing/mask))
```
Comment (francinum) [inline code/game/machinery/iv_drip.dm]: ?

**Comment by That0nePerson** [comment]
oh yeah also, plasmamen can also use this pretty well if they lose their smaller plasma tank

**Comment by francinum** [comment]
I'd prefer the glovebox be melded with paper bin code, but that's a wish not a necessity.

**Review by francinum — APPROVED** [review]

**Pull request merged**